frontend/app.ts

```diff
@@ -12,26 +12,34 @@ declare global {
 
 export type BlockProps = Omit<BaseBlockProps, "iframeUrl">;
 
-export function renderBlock(containerId: string, props: BlockProps) {
+export function renderBlock(containerElement: HTMLElement | null, props: BlockProps) {
```

Comment:
Can we get away without having to specify `null` here?

Reply:
Good catch, fixed.